logic.py

```diff
@@ -6,17 +6,26 @@
 
 
 def workflow_state_message(workflow, task_breadcrumbs, new_decision_action_id):
```

Comment:
It's only cosmetic but you could name these slightly more consistently?
```
new_task_id
previous_task_id
new_task_breadcrumbs
previous_task_breadcrumbs
```
or...
```
new_decision_action_id
previous_decision_action_id
new_action_breadcrumbs
previous_action_breadcrumbs
```